core/tests/protractor_utils/editor.js

```diff
@@ -1199,6 +1200,7 @@ var acceptSuggestion = function(suggestionDescription) {
       matchingSuggestionRows[0].click();
       general.waitForSystem();
       element(by.css('.protractor-test-view-suggestion-btn')).click();
+      general.waitForSystem();
```

Comment:
@nithusha21: Can we remove this statement now since we do not require this? What do you think? Thanks!

Reply:
Actually I'm not sure the second wait helped either (It failed the first time :( ). Might need more investigation to find out why this fails.

That being said, I feel this wait is actually necessary -- It did fail at this location twice for me locally. I feel there are multiple flaky points in this test 😅) Can you also help out to find if any other location is flaky. Thanks!

Reply:
I did try this test out locally. It passes for me. I am not too sure about multiple flaky points in the test. What should be the right action in terms of this PR -- shall this be merged or should this be kept on hold?
Now that the tests pass, I feel that this PR can go in and then we could see regarding the re-occurrence of this flakiness. What do you suggest?
Thanks!

Reply:
Yes that makes sense. We can try out this solution and see if the flakiness persists. If so I'll try investigating further. But for now let's merge this as I feel it should reduce the flake :)